sympy/physics/hep/gamma_matrices.py

```diff
@@ -340,7 +340,8 @@ def kahane_simplify(expression):
     Parameters
     ==========
 
-    `expression`    the tensor expression containing the gamma matrices to simplify.
+    ``expression``:
+        The tensor expression containing the gamma matrices to simplify.
```

Comment:
The error is something in this docstring. I'm not sure what the problem is. Should there be a blank line here?

Reply:
Oh perhaps the issue is there should be a space before the colon after ``` ``expression`` ```

Reply:
Changed.